docs/controls/HamburgerMenu.md

```diff
@@ -143,10 +143,9 @@ namespace HamburgerSample
         }
     }
 }
-
 ```
 
-## <a name="navview"></a> Moving to NavigationView
```

Comment:
there are external links that would be broken with this change, please revert back